source/_posts/2017-02-04-babyphone.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Baby monitoring"
+description: "Make your Home fit for baby monitoring."
+date: 2016-07-28 06:00:00 +0200
+date_formatted: "July 28, 2016"
+author: Pascal Vizeli
+comments: true
+categories: How-To Babyphone
+og_image: /images/blog/2017-02-babyphone/social.png
+---
+
+For freshly baked parents comes quickly the question how to monitor the sleep of his child. We want to be awakened when something is. There are many expensive solutions and thus other devices which stand only for a purpose somewhere. What is if we could build a super babephone with little means? With Home-Assistant is this quite simply and possible! We use our existing e.g. Sonos as a loudspeaker and in the night we want to supposed a dimmed lit on the way to the children's room. I send also a notification with a image to my phone, so I have a nice history on next day.
+
+The whole solutions goes also for other purposes, e.g. the audio of the surveillance camera should transfer on a noise into our apartment.
+
+<!--more-->
+
+We need an IP camera in the children's room with sound. This is the simplest solution and we describe this here in the blog. It is also possible to equip a Raspberry with a microphone and send the sound to our Home-Assistant with `ffmpeg -f alsa -i hw:1,0 -vn -f rtp rtp://236.0.0.1:2000` over multicast.
```

Comment:
I think this command `ffmpeg -f alsa -i hw:1,0 -vn -f rtp rtp://236.0.0.1:2000` needs some clarification. What is the IP address? Of HA?